src/components/PlatformNavigator/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+import { Button, Flex, Text } from '@aws-amplify/ui-react';
+import { IconChevron } from '@/components/Icons';
+import { frameworks } from '@/constants/frameworks';
+import { InfoPopover } from './InfoPopover';
+import Link from 'next/link';
+
+export function PlatformNavigator({ currentPlatform }) {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const platformItem = frameworks.filter((platform) => {
+    return platform.title === currentPlatform;
+  })[0];
+
+  return (
+    <>
+      <nav aria-labelledby="platformBtn" className={`platform-navigator`}>
+        <Text fontWeight={"bold"}>Choose your framework/language</Text>
+        <Flex alignItems="center">
```

Comment:
Can we add some spacing between this Text and Flex so Choose your framework doesn't sit right against the button below it?